policyengine_us/variables/gov/states/ky/tax/income/exclusions/pension_income/ky_service_credit_months_pre_cutoff.py

```diff
@@ -0,0 +1,9 @@
+from policyengine_us.model_api import *
+
+
+class ky_service_credit_months_pre_cutoff(Variable):
```

Comment:
ideally this would be a policy parameter and we'd apply a panel structure but for now let's clarify the hard-coding
```suggestion
class ky_service_credit_months_pre_1998(Variable):
```